connexion/decorators/security.py

```diff
@@ -12,6 +12,8 @@
 from ..exceptions import (ConnexionException, OAuthProblem,
                           OAuthResponseProblem, OAuthScopeProblem)
 
+from http.cookies import SimpleCookie
```

Comment:
you'll need to use six.http_cookies since we support python 2.7

Reply:
Thank you!
I have done it.
It works as I can see.